pontoon/settings/base.py

```diff
@@ -58,6 +58,13 @@ def path(*args):
 # tests.
 SSLIFY_DISABLE = DEBUG or os.environ.get('CI', False)
 
+# URL to the RabbitMQ server
+BROKER_URL = os.environ.get('RABBITMQ_URL', None)
+
+# Execute celery tasks locally instead of in a worker unless the
+# environment is configured.
+CELERY_ALWAYS_EAGER = os.environ.get('CELERY_ALWAYS_EAGER', 'False') != 'False'
```

Comment:
`bool('False') == True`, and environment variables are always strings, so we can't use numbers or primitives.

Long term this is still weird, though. A reorganization of the settings using django-configurations has been floating around my head as a someday task for a while.